### PR TITLE
BaSP-TG-46: add and edit Timesheet Page

### PR DESCRIPTION
**Pull request by laura-brussa12**
- Create form for Timesheet Page 
- Add feature
- Edit feature

### CODE REVIEW

File: src/Components/TimeSheet/index.js [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+            </option>
+            {tasksOptions.map((option) => (
+              <option key={option.value} value={option.value}>
+                {option.label}
+              </option>
+            ))}
+          </select>
+        </div>
+        {/* hacer el map de las tareas del usuario */}
+        <div className="input">
```
Comment (GuidoCerioni) [inline src/Components/TimeSheet/index.js:172]: I would add a label or a placeholder to tell the user what to put on each input

**Review by GuidoCerioni — COMMENTED** [review]
I found it difficult to guess what to put on each input, but everything works fine!

**Review by FiolAxel — APPROVED** [review]

File: src/Components/TimeSheet/index.js [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+      const tasks = await tasksData.json();
+      for (let i = 0; i < tasks.data.length; i++) {
+        setTasksOptions((tasksOptions) => [
+          ...tasksOptions,
+          {
+            value: `${tasks.data[i]._id}`,
+            label: `${tasks.data[i].description}`
+          }
+        ]);
+      }
```
Comment (franco-marini) [inline src/Components/TimeSheet/index.js:68]: Review this logic again. You could use a `.map` function to set the task options. Could you review this with a partner?

**Review by HiginiaMed — APPROVED** [review]
Great job!

**Comment by tobiaspujol** [comment]
I can't see any data on timesheets

**Comment by laura-brussa12** [comment]
@tobiaspujol I just look and I can see the data on timesheets... Remember that the URL is timesheets/form

**Comment by tobiaspujol** [comment]
You're right. It's ok!

File: src/Components/TimeSheet/index.js [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+      options.method = 'PUT';
+      url = `${process.env.REACT_APP_API_URL}/time-sheets/${timeSheetId}`;
+    }
+    try {
+      const response = await fetch(url, options);
+      const data = await response.json();
+      if (data.error) {
+        setModalMessage('There was an error');
+      }
+      console.log(data);
```
Comment (AlejandroSantini) [inline src/Components/TimeSheet/index.js:128]: You should delete console.log

**Review by AlejandroSantini — APPROVED** [review]

**Pull request merged**